pkg/metricsservice/client.go

```diff
@@ -37,17 +37,22 @@ type GrpcClient struct {
 }
 
 func NewGrpcClient(url, certDir, authority string) (*GrpcClient, error) {
-	defaultConfig := `{
-		"methodConfig": [{
-		  "timeout": "3s",
-		  "waitForReady": true,
-		  "retryPolicy": {
-			  "InitialBackoff": ".25s",
-			  "MaxBackoff": "2.0s",
-			  "BackoffMultiplier": 2,
-			  "RetryableStatusCodes": [ "UNAVAILABLE" ]
-		  }
-		}]}`
+	defaultConfig := fmt.Sprintf(`{
+			"methodConfig": [{
+			"timeout": "3s",
+			"waitForReady": true,
+			"retryPolicy": {
+				"InitialBackoff": ".25s",
+				"MaxBackoff": "2.0s",
+				"BackoffMultiplier": 2,
+				"RetryableStatusCodes": [ "UNAVAILABLE" ]
+			}
+			}],
+			"healthCheckConfig": {
+				"serviceName": "%s"
```

Comment:
I _think_ this will break if someone uses a 2.14 keda-metrics-apiserver with a <2.14 keda-operator, but I don't think this is a supported scenario, right? So is this ok or do we need to document this possible break

Reply:
That scenario won't happen as KEDA is released/used "at once". Maybe during the upgrading process the metrics server could miss the connectivity with the operator during some seconds due to the mismatch, but it's not something that will happen normally.

Reply:
yeah, not an issue